frontend/javascripts/oxalis/view/action-bar/starting_job_modals.tsx

```diff
@@ -503,8 +491,78 @@ function RunAiModelTab({ aIJobModalState }: { aIJobModalState: string }) {
   );
 }
 
+function AlignmentTab() {
+  const centerImageStyle = {
+    margin: "auto",
+    width: 150,
+  };
+  return (
+    <div>
+      <Space align="center">
+        <Radio.Button className="aIJobSelection" checked={true}>
+          <Card bordered={false}>
+            <Space direction="vertical" size="small">
+              <Row className="ai-job-title">Align Sections</Row>
+              <Row>
+                <img
+                  src={`/assets/images/${jobNameToImagePath.align_sections}`}
+                  alt={"Example of improved alignment of slices"}
+                  style={centerImageStyle}
+                />
+              </Row>
+            </Space>
+          </Card>
+        </Radio.Button>
+      </Space>
+      <AlignSectionsForm />
+    </div>
+  );
+}
+
+function UseSkeletonFormItem() {
+  const tracing = useSelector((state: OxalisState) => state.tracing);
+  return (
+    <div>
+      <Form.Item
+        name="useAnnotation"
+        label={
+          <Space>
+            <div style={{}}>
+              Manual Matches
```

Comment:
A forced space is needed here:
```suggestion
              Manual Matches{" "}
```
Otherwise there is no space between the text and the icon which doesn't look nice :)